site/docs-md/guides/ionic-framework-app.md

```diff
@@ -0,0 +1,250 @@
+
+## Required Dependencies
+Building and deploying iOS and Android apps require additional dependencies, including an iOS and Android device. Please [follow the instructions here](../getting-started/dependencies) before continuing.
+
+## Prepare an Ionic App
+If you have an existing Ionic app, skip this section. If not, let's create an Ionic app first. In a Terminal, install Ionic (this also updates to the latest version of Ionic):
+
+```bash
+npm install -g ionic
+```
+
+Next, create a new Ionic app based on the "tabs" starter project:
+
+```bash
+ionic start capApp tabs
+```
+
+## Add Capacitor
+Capacitor was designed to drop into any existing modern JavaScript web app - Ionic included!
+
+```bash
+cd capApp
+npm install @capacitor/core @capacitor/cli
+```
+
+Next, initialize Capacitor with your app information.
+
+```bash
+npx cap init
+```
+
+**App name:** CapApp  
+**App Package ID:** com.example.capapp
+
+## Build the App Once
+Before adding any native platforms to this project, the app must be built at least once. A web build creates the web assets directory that Capacitor needs (`www` folder in Ionic projects):
+
+```bash
+ionic build
+```
+
+Next, add any platforms (besides the web, of course) that you'd like to build for:
+
+```bash
+npx cap add ios
+npx cap add android
+```
+
+Upon running these commands, both `android` and `ios` folders at the root of the project are created. These are entirely separate native project artifacts that should be considered part of your Ionic app (i.e., check them into source control).
+
+## Adding Camera Functionality
+Next up, we'll add the ability to take photos with the device's camera using the Capacitor [Camera API](../apis/camera).
+
+To begin, open the Tab2 HTML page under the `src/app/tab2` folder. Add an image tag that will display the current photo taken with the camera and add an Ionic fab button that will open the camera when clicked:
+
+```html
+<ion-content>
+  <img [src]="photo" >
+
+  <ion-fab vertical="bottom" horizontal="center" slot="fixed">
+    <ion-fab-button (click)="takePicture()">
+      <ion-icon name="camera"></ion-icon>
+    </ion-fab-button>
+  </ion-fab>
+</ion-content>
+```
+
+Next, we need to add the logic that will power the camera functionality. Open `src/app/tab2/tab2.page.ts`  and import the Capacitor Plugins and Camera classes:
+
+```typescript
+import { Plugins, CameraResultType, CameraSource } from '@capacitor/core';
+```
+
+Next, implement the `takePicture()` method:
+
+```typescript
+export class Tab2Page {
+  async takePicture() {
+    const image = await Plugins.Camera.getPhoto({
+      quality: 100,
+      allowEditing: false,
+      resultType: CameraResultType.Base64,
+      source: CameraSource.Camera
+    });
+
+    this.photo = this.sanitizer.bypassSecurityTrustResourceUrl(image && (image.base64Data));
```

Comment:
After changing the resultType  to `CameraResultType.DataUrl`, you'll get it with `image.dataUrl` instead of base64Data.